.github/workflows/antsibull-build.yml

```diff
@@ -54,6 +54,8 @@ jobs:
         run: |
           python3 -m pip install --upgrade pip
           python3 -m pip install poetry ansible-core
+          poetry install
+          poetry update
```

Comment:
Was the plan not to get rid of poetry entirely ? Should this be a pip install ``path/to/antsibull/src`` instead of poetry ?